.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
       - name: Run tests
-        run: bundle exec rake --rakefile Rakefile_ci -T | grep release
+        run: bundle exec rake --rakefile Rakefile_ci -T | grep --regexp 'rake strings' --regexp 'rake release' --quiet
```

Comment:
The name does not correspond to what the run command does…  We change it to something like "check that required tasks are available" maybe?

Reply:
updated it